src/components/views/auth/AuthBody.tsx

```diff
@@ -17,6 +17,16 @@ limitations under the License.
 import React from 'react';
 
 export default class AuthBody extends React.PureComponent {
+    componentDidMount() {
+        const unloadCallback = (event) => {
+            event.preventDefault();
```

Comment:
This could do with a comment saying that this is to trigger a confirmation dialog

Reply:
noted